source/content/guides/switch-drupal-recommended-upstream.md

```diff
@@ -10,11 +10,11 @@ contributors: [kporras07]
 reviewed: "2022-04-03"
```

Comment:
This guide should now support both drupal-project and drupal-recommended as the starting point

Reply:
Also, not sure if we want to preserve/update the title and url. If changing the URL, I'm not sure about the policy for redirects. Maybe a question for docs team.

Reply:
I've updated this doc in [47c7142](https://github.com/pantheon-systems/documentation/pull/7292/commits/47c7142258b86e51a2d9e13fd671060c199dd8c4) to include both drupal-project and drupal-recommended. Also changed the title to `Switch from Drupal 9 to Drupal (Composer Managed) Upstream` to accurately reflect the new upstream name.

@pantheondocs Would it be possible to change the URL of this page from https://pantheon.io/docs/guides/switch-drupal-recommended-upstream to. https://pantheon.io/docs/guides/switch-drupal-composer-managed-upstream, and put a redirect in place for that?